app/assets/javascripts/mapknitter/Map.js

```diff
@@ -462,6 +462,28 @@ MapKnitter.Map = MapKnitter.Class.extend({
     if (this.editing._mode != "lock") e.stopPropagation()
   },
 
+  synchronizeData: function(warpables) {
```

Comment:
The updation corresponding to each change will happen with the help of this function. This function gets all the necessary information related to the warpables on the map (which are broadcasted my the server on the concurrent_editing_channel)